extensions/wikia/ImageReview/ImageReviewSpecialController.class.php

```diff
@@ -191,16 +277,16 @@ protected function getStatsPageTitle() {
 	}
 
 	protected function parseImageData($data) {
-		$images = array();
+		$images = [];
 
 		foreach( $data as $name => $value ) {
-			if (preg_match('/img-(\d*)-(\d*)/', $name, $matches)) {
+			if ( preg_match( '/img-(\d*)-(\d*)/', $name, $matches ) ) {
 				if ( !empty($matches[1]) && !empty($matches[2]) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)